package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "collective.js.throttledebounce",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "git@github.com:collective/collective.js.throttledebounce.git",
+  "author": "yarn add -D requirejs chokidar-cli",
+  "license": "GPL2",
+  "private": true,
+  "scripts": {
+    "build": "r.js -o build.js",
+    "build-js": "r.js -o build.js",
+    "watch": "chokidar \"./ftw/slider/browser/resources/(*.js|*.css)\" -i \"**/*-compiled.(js|css)\" -c \"/bin/sh -c \\\"if [[ '{path}' = *.js ]]; then r.js -o build.js; else r.js -o buildcss.js; fi;\\\"\""
```

Comment:
You could replace the command with a simple `yarn build` and remove the watch of `*.css` files.

Reply:
hmm, I think it should work now and it's like we do it normally, so I'd stick with it 😉